src/app/app.js

```diff
@@ -1,32 +1,58 @@
 import express from 'express';
+import cookieSession from 'cookie-session';
 import helmet from 'helmet';
 import pinoMiddleware from 'express-pino-logger';
 import compression from 'compression';
 import staticGzip from 'express-static-gzip';
+import fetch from 'node-fetch';
+
+import auth from '../auth';
 import home from '../home';
 import orgs from '../orgs';
 import {pageNotFoundMiddleware, internalServerErrorMiddleware} from '../errors';
 import csp from './app.csp';
 
-export default function ({logger} = {}) {
+export default function (config) {
   const app = express();
 
-  if (logger) {
-    app.use(pinoMiddleware({logger}));
-  }
+  app.use(pinoMiddleware(config.logger));
+
+  app.use(cookieSession({
+    name: 'pazmin-session',
+    keys: [config.sessionSecret],
+    secure: config.isProduction,
+    httpOnly: true
+  }));
 
   app.use('/assets', staticGzip('dist/assets', {immutable: true}));
   app.use(compression());
 
   app.use(helmet());
   app.use(helmet.contentSecurityPolicy(csp));
 
-  app.use('/orgs', orgs);
   app.use('/', home);
```

Comment:
I think the `home` route should be behind the authentication ... we have no need for any "unauthenticated" routes yet

Reply:
This is really bad excuse, but: Moving this, will fail the tests. We should probably fix that by coming up with a better way of handling the token in tests. But I don't want to block this story on that. Although it introduces this problem, the idea of homepage is still undecided. Let's cross that bridge when we get to it?